sympy/utilities/lambdify.py

```diff
@@ -50,9 +50,9 @@
     "E": "e",
     "I": "j",
     "ln": "log",
-    #"lowergamma":"lower_gamma",
+    "lowergamma":"gammainc",
```

Comment:
it doesn't give any error if I comment out the above two lines , so we can remove them